content/sensu-go/5.21/reference/license.md

```diff
@@ -55,7 +55,6 @@ Error: not found
 ## Entity limit
 
 Your commercial license may include the entity limit and entity class limits tied to your Sensu licensing package.
-An entity limit of `0` allows unlimited entities.
```

Comment:
This still applies, but if it no longer makes sense to mention, that is ok!

Reply:
I just thought it might be confusing since the response header shows `/unlimited` -- I'm not sure where they would see the `0`. Is it in the license itself? (mine doesn't show any entity limit, so I can't quite tell)

Reply:
I can't find it in the skunkworks licenses either, so I'm pretty sure I haven't looked in the right place.